src/components/Pages/todosPage.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  FC, useContext, useEffect, useMemo, useState,
+} from 'react';
+import { getTodos } from '../../api/todos';
+import { ErrorTodo } from '../../types/ErrorTodo';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { AppContext } from '../AppContext';
+import { ErrorMessage } from '../errorMessage';
+import { Footer } from '../Footer';
+import { Header } from '../Header';
+import { TodoList } from '../TodoList';
+
+type Props = {
+  user: User | null,
+};
+
+export const TodosPage: FC<Props> = ({ user }) => {
+  const {
+    closeErrorMessage,
+    errorTodo,
+    todosFromServer,
+    setTodosFromServer,
+    showErrorMessage,
+    timerId,
+  } = useContext(AppContext);
+  const [newTodo, setNewTodo] = useState<Todo | null>(null);
+
+  const createTodo = async (title: string) => {
+    setNewTodo({
+      id: 0,
+      title,
+      completed: false,
+      userId: user?.id,
+    });
+  };
+
+  async function loadTodos() {
+    closeErrorMessage();
+
+    if (user) {
+      const loadedTodos = await getTodos(user.id);
+
```

Comment:
```suggestion
    if (!user) {
      return
    }
```
так можна зменшити вкладеність